src/app.js

```diff
@@ -19,7 +19,7 @@ const app = express();
 const PORT = 3000;
 
 //* 소셜 로그인 설정
-passportConfig(app);
+// passportConfig(app);
```

Comment:
소셜로그인설정을 왜 주석처리하셨나요?

```suggestion
 passportConfig(app);
```

Reply:
전에. env 파일 때문에 주석 처리한다고 하셨던 거 같은데 수정해 주세요!